### ISSUE TITLE
[enhancement]Convert morph group to shapekeys

### ISSUE THREAD

**Issue by NaughtDZ** [MMD-Blender/blender_mmd_tools#168]
In fact, most 'morph groups' themselves are only composed of shapekeys that transform multiple vertices.
So, can we add a feature to convert "morph group" into a shapekey with just one click, that is, automate the process : setting a deformation to 1 in the "morph group" and then selecting " the merge deformation to a new shape key "on the grid shape key then rename it  .While,this funciton can ignoring material or bone changes,so there may no have errors.

**Comment by rintrint** [comment]
Yes, but do you need this feature frequently? If so, could you share more about your use case and why it's important for your workflow?

The model author separates them, there must be a reason.
